ocs_ci/deployment/helpers/lso_helpers.py

```
@@ -74,10 +78,19 @@ def setup_local_storage(storageclass, add_new_disks=True):
 
         storage_node_names = get_compute_node_names(no_replace=True)
 
-        if config.ENV_DATA.get(
-            "odf_provider_mode_deployment", False
-        ) and config.ENV_DATA.get("mark_masters_schedulable", True):
-            storage_node_names.extend(get_master_nodes())
+        provider_mode = config.ENV_DATA.get("odf_provider_mode_deployment", False)
+        mark_schedulable = config.ENV_DATA.get(
+            "mark_masters_schedulable", True if provider_mode else False
```

Ruff (0.16.0)

[warning] Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)